mne/bem.py

```diff
@@ -1667,14 +1688,15 @@ def make_flash_bem(subject, overwrite=False, show=True, subjects_dir=None,
     outer skin) from multiecho FLASH MRI data with spin angles of 5 and 30
     degrees, in mgz format.
 
-    This function assumes that the flash images are available in the
-    folder mri/bem/flash within the freesurfer subject reconstruction.
+    This function assumes that the flash images are available in the current
+    folder.
```

Comment:
This docstring wasn't correct. This function is called by command ``make_flash_bem`` so I don't want to change the behavior of the function.